editor/ImportExport/readme.md

```diff
@@ -1,34 +1,34 @@
+The sample also [increases the max SignalR message size](https://docs.telerik.com/blazor-ui/components/editor/overview#large-content-support) in `Startup.cs`.
 
 ## Import
 
-The conversion happens in the `FileConverter.cs` file - the `GetHtmlString` method.
+The conversion happens in the `GetHtmlString` method in `FileConverter.cs`.
 
-In this sample, it reads a `docx` file in the `wwwroot` folder, but you can develop it further to let the user upload a file before reading it, and to use other formats (such as RTF).
+The sample reads a DOCX file from the `wwwroot` folder, but you can enhance it to let users upload a file, or import other formats.
 
-A key point is that the Editor needs only the body of the content - full HTML cannot be edited, so you must make sure to remove the `html`, `head` elements, and to only provide the contents of the `body` tag without the tag itself. In this sample, the conversion is done as a `Fragment` export level, which keeps only the body and adds inline styles - this is a feature of the Telerik DPL. If you use the full page export, by default styles go into a separate `<style>` tag and you should consider either including that (it might affect the rest of the page), or using inline styles in the export settings.
+A key point is that the Editor needs only the `<body>` of the imported HTML content. Make sure to remove the `<html>` and `<head>` elements, and only provide the contents of the `<body>` without the tag itself. In this sample, the conversion is done as a `Fragment` export level, which keeps only the body and adds inline styles - this is a feature of Telerik Document Processing. If you use the full page export, by default styles go into a separate `<style>` tag and you should consider either including that (it might affect the rest of the page), or using inline styles in the export settings.
 
-The `ReadFile` helper function shows a sample way of tackling various formats and creating appropriate providers for them to import.
+The `ReadFile` helper function shows how to handle various formats and create appropriate providers for them to import.
 
-Once the HTML string is prepared, we simply return it in the view-model and pass it to the editor `Value`.
+Once the HTML string is prepared, we pass as the Editor `Value`.
 
 ## Export
 
-The export is done in the in the `FileConverter.cs` file - the `ExportAndDownloadHtmlContent` method.
+The export happens in the `ExportAndDownloadHtmlContent` method in `FileConverter.cs`.
 
-First, it creates the in-memory document with the HTML that we need for the conversion, then checks what provider to use based on the file exetension, and then - with the use of a helper class (`FileDownloader.cs`) and a JavaScript function (in `fileDownloader.js`) - it downloads that content in the browser.
+1. It creates the in-memory document with the HTML.
+1. It checks what provider to use based on the file exetension.
```

Comment:
```suggestion
1. It checks what provider to use based on the file exetension.
1. It adds an image converter that will convert any non-JPG images to JPG. Otherwise the exporting will fail with an exception.
```